### PR TITLE
SPARK-781: Log the temp directory path when Spark says "Failed to create temp directory."

### PR DESCRIPTION
**Pull request by rxin**

### CODE REVIEW

File: core/src/main/scala/spark/Utils.scala [hunk 1]
```diff
@@ -116,8 +116,8 @@ private object Utils extends Logging {
     while (dir == null) {
       attempts += 1
       if (attempts > maxAttempts) {
-        throw new IOException("Failed to create a temp directory after " + maxAttempts +
-            " attempts!")
+        throw new IOException("Failed to create a temp directory under (" + root + ") after " +
```
Comment (mateiz) [inline core/src/main/scala/spark/Utils.scala]: Shouldn't this say (under DIR) instead of under (DIR)?

**Comment by rxin** [comment]
Good find - PTAL.

**Comment by mateiz** [comment]
It looks good, but let's wait for Jenkins to test it. Jenkins, this is ok to test.

**Comment by mateiz** [comment]
Looks good, thanks.

**Pull request merged**